src/api.v2/routes/access.js

```diff
@@ -0,0 +1,13 @@
+const {
+  getExperimentUsers,
+} = require('../controllers/accessController');
+
+const { expressAuthenticationOnlyMiddleware, expressAuthorizationMiddleware } = require('../middlewares/authMiddlewares');
+
+module.exports = {
+  'access#getExperimentUsers': [
+    expressAuthenticationOnlyMiddleware,
+    expressAuthorizationMiddleware,
```

Comment:
I believe the `expressAuthorizationMiddleware` already includes authentication

Reply:
oh right!